src/SdfEntityCreator.cc

```diff
@@ -788,8 +794,39 @@ Entity SdfEntityCreator::CreateEntities(const sdf::Visual *_visual)
   // \todo(louise) Populate with default material if undefined
   if (_visual->Material())
   {
+    sdf::Material visualMaterial = *_visual->Material();
+    if (!_visual->Material()->ScriptUri().empty())
+    {
+      gzwarn << "Gazebo does not support Ogre material scripts. See " <<
+      "https://gazebosim.org/api/sim/8/migrationsdf.html#:~:text=Materials " <<
```

Comment:
does the `#:~:text=Materials` part work? It takes me to the migration page but not the Materials section.

Reply:
Its using [text fragments](https://developer.mozilla.org/en-US/docs/Web/Text_fragments#simple_text_fragment_with_textstart), and mostly browser dependent. For me, it highlights and scrolls to the Material section but scrolls back up the page. Do you think it would be useful?

Reply:
ah ok I thought it would jump to that section. Yes I see it highlighted